htdocs/GetCSS.php

```diff
@@ -27,11 +27,8 @@
 require_once __DIR__ . "/../vendor/autoload.php";
 // Ensures the user is logged in, and parses the config file.
 require_once "NDB_Client.class.inc";
-$client = new NDB_Client();
-
-if ($client->initialize(null) == false) {
-    return false;
-}
+$client    = new NDB_Client();
+$anonymous = ($client->initialize("../project/config.xml") === false);
```

Comment:
$client->initialize always return true.
$anonymous should be removed.

Reply:
should I make the same change in GetJS.php?

Reply:
$client->initialize shouldn't always return true, it should only return true if the user is authenticated

Reply:
my mistake.